regression-test/suites/datatype_p0/float/test_float_dft_val.groovy

```diff
@@ -0,0 +1,33 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
```

Comment:
In mysql,use these sql:
 create table t2 (
        `k1` int,
        `k2` float default "3.14"
    );

insert into t2 values ();
insert into t2 values ();
select * from t2;
select * from t2 where k2 = 3.14;



k1 | k2
-- | --
null | 3.14
null | 3.14